ui/components/multichain/multichain-connected-site-menu/multichain-connected-site-menu.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { useSelector } from 'react-redux';
+import {
+  BackgroundColor,
+  BorderColor,
+  BorderRadius,
+  IconColor,
+  Size,
+} from '../../../helpers/constants/design-system';
+import { BadgeWrapper, Icon, ICON_NAMES } from '../../component-library';
+import Box from '../../ui/box';
+import { getSelectedIdentity } from '../../../selectors';
+import Tooltip from '../../ui/tooltip';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export const MultichainConnectedSiteMenu = ({
+  className,
+  globalMenuColor,
+  status,
+  text,
+}) => {
+  const t = useI18nContext();
+  const selectedAccount = useSelector(getSelectedIdentity);
+  return (
+    <Box
+      className={classNames('multichain-connected-site-menu', className)}
+      data-testid="connection-menu"
+    >
+      <Tooltip
+        title={
+          status === 'STATUS_NOT_CONNECTED'
+            ? t('statusNotConnectedAccount')
+            : `${selectedAccount?.name} ${text?.toLowerCase()}`
+        }
+        data-testid="multichain-connected-site-menu__tooltip"
+        position="bottom"
+      >
+        <BadgeWrapper
+          positionObj={
+            status === 'STATUS_CONNECTED_TO_ANOTHER_ACCOUNT'
```

Comment:
Do we have a constant for `STATUS_CONNECTED_TO_ANOTHER_ACCOUNT`?

Reply:
Done [90affbd](https://github.com/MetaMask/metamask-extension/pull/18167/commits/90affbd1e63dec216a1c7edd2445c6839c584794)